.github/workflows/shiny-deploy.yaml

```diff
@@ -0,0 +1,48 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# see https://www.tidyverse.org/blog/2022/06/actions-2-0-0/ for info on setup-r-dependencies@v2
+on:
+  push:
+    branches: [production]
```

Comment:
I setup two github environments. One for staging and one for production. Staging is deployed from "master" and "production" to "production"

Could you could change this to `[master, production]`?

Reply:
I have pushed a new file. I haven't experimented with multiple GHA environments yet!